src/page-building/career/career-page.tsx

```diff
@@ -14,7 +14,6 @@ import { useTranslation } from 'gatsby-plugin-react-i18next';
 import { SyPersonioJob } from '../../types';
 
 const Paragraph = styled.p`
-  ${TextStyles.textR}
```

Comment:
Thanks for removing this. The following styles (space between adjacent paragraphs) is not needed as well. The template for this component only contains a single `<Paragraph>` so there is no point in keeping these styles. I suggest to completely remove this `Paragraph` and instead use a simple `<p>` in the template.

Reply:
I am not sure, but I think the Plan was to add multiple paragraphs.
For now there is only one, so I left it out. If we add more, we have to change it again, but I think it is less enough to remove it for now.

Reply:
I have removed those paragraphs in some PR before. While this change is okay I'm worrying about text paragraphs in general. We are not capable of freely writing texts right now 😕 but I think we can talk about that in https://github.com/satellytes/satellytes.com/issues/251